simpl/threadlocal.py

```diff
@@ -0,0 +1,52 @@
+"""Threadlocal."""
```

Comment:
config.py has a long description with examples of how to use it. That woud be a good pattern to continue in simpl; to have each module documented in the __doc__.